lib/chef/http.rb

```diff
@@ -154,6 +154,25 @@ def request(method, path, headers={}, data=false)
       raise
     end
 
+    # Makes an HTTP request to +path+ with the given +method+, +headers+, and
+    # +data+ (if applicable). Does not apply any middleware, besides that
+    # needed for Authentication.
+    def raw_request(method, path, headers={}, data=false)
```

Comment:
I think this either needs to go one level deeper in the class hierarchy (i.e., on `ServerAPI`) or maybe in a `BasicServerAPI` which only includes the necessary middleware. Regular `Chef::HTTP` shouldn't know about any middleware stuff at all and should only make vanilla HTTP requests.